plugin/fzf.vim

```diff
@@ -972,21 +972,47 @@ else
 endif
 
 function! s:popup(opts) abort
+  let create_opts = copy(a:opts)
+
+  " Remove known FZF options
+  let xoffset = 0.5
+  let yoffset = 0.5
+
+  if has_key(a:opts, 'xoffset')
+    let xoffset = a:opts.xoffset
+    call remove(create_opts, 'xoffset')
+  endif
+
+  if has_key(a:opts, 'yoffset')
+    let yoffset = a:opts.yoffset
+    call remove(create_opts, 'yoffset')
+  endif
+
+  if has_key(a:opts, 'border')
+    call remove(create_opts, 'border')
+  endif
+
```

Comment:
Can we also remove `highlight` property? It's deprecated but some users may still have it.

https://github.com/junegunn/fzf/commit/6e93eefc82d8fb749e97d094e6ad6fbbe09d289b

Reply:
```vim
  " Remove known FZF options
  let xoffset = get(a:opts, 'xoffset', 0.5)
  let yoffset = get(a:opts, 'yoffset', 0.5)

  for key in ['xoffset', 'yoffset', 'border', 'highlight']
    if has_key(a:opts, key)
      call remove(create_opts, key)
    endif
  endfor
```

What do you think?

Reply:
Looks good! I'll add a commit for it